pkg/webhookconfig/monitor.go

```diff
@@ -106,6 +107,12 @@ func (t *Monitor) Run(register *Register, certRenewer *tls.CertRenewer, eventGen
 				t.log.Error(err, "")
 			}
 
+			// update namespaceSelector every 30 seconds
+			if register.autoUpdateWebhooks {
+				logger.V(3).Info("updating webhook configurations for namespaceSelector with latest kyverno ConfigMap")
+				configData.UpdateWebhook()
```

Comment:
Can we use `register.UpdateWebhookChan` to trigger the webhook updates? This is the same channel https://github.com/kyverno/kyverno/blob/b6447e06496c21d114a636c3d92e4b3f3d8dc40a/cmd/kyverno/main.go#L267.

Reply:
> Can we use `register.UpdateWebhookChan` to trigger the webhook updates? This is the same channel

Right! I didn't notice this, thanks for pointing it out. Will revert the newly created methods for updating webhook cfgs and reconciling policy report as well as the `configData` param.